bindings/build.rs

```diff
@@ -0,0 +1,187 @@
+#[macro_use]
+extern crate thiserror;
+
+extern crate serde;
+extern crate serde_json;
+
+fn main() -> webview2_nuget::Result<()> {
+  let package_root = webview2_nuget::install()?;
+  webview2_nuget::update_windows(package_root)?;
+
+  windows::build!(
+      Microsoft::Web::WebView2::Core::*,
+      Windows::Foundation::*,
+      Windows::Storage::Streams::*,
+      Windows::Win32::Com::*,
+      Windows::Win32::DisplayDevices::{
+        POINT,
+        POINTL,
+        RECT,
+        SIZE
+      },
+      Windows::Win32::Gdi::UpdateWindow,
+      Windows::Win32::HiDpi::{
+        PROCESS_DPI_AWARENESS,
+        SetProcessDpiAwareness
+      },
+      Windows::Win32::KeyboardAndMouseInput::SetFocus,
+      Windows::Win32::MenusAndResources::HMENU,
+      Windows::Win32::Shell::{
+        DragFinish,
+        DragQueryFileW,
+        HDROP,
+        ITaskbarList,
+        TaskbarList
+      },
+      Windows::Win32::SystemServices::{
+        BOOL,
+        CLIPBOARD_FORMATS,
+        DRAGDROP_E_INVALIDHWND,
+        DV_E_FORMATETC,
+        GetCurrentThreadId,
+        GetModuleHandleA,
+        HINSTANCE,
+        LRESULT,
+        PWSTR,
+        userHMETAFILEPICT,
+        userHENHMETAFILE
+      },
+      Windows::Win32::WindowsAndMessaging::*
+  );
+
+  Ok(())
+}
+
+mod webview2_nuget {
+  use std::{convert::From, env, fs, io, path::PathBuf, process::Command};
+
+  const WEBVIEW2_NAME: &str = "Microsoft.Web.WebView2";
+  const WEBVIEW2_VERSION: &str = "1.0.824-prerelease";
```

Comment:
Looks like the library structure is different from the previous stable release (1.0.774.44).
Will the future stable release be the same as the current one?

Reply:
I see. It should be fine we use the prerelease package.

Reply:
Do we have to ship the dlls forever? Or just until the current prerelease becomes the evergreen stable? Because I really don't want to ship that by default forever, and

Reply:
So I think the prerelease packages are a superset of what will eventually show up in the stable/release packages. I expect the paths/structure of the WinRT APIs to match at that point.

Reply:
> Do we have to ship the dlls forever? Or just until the current prerelease becomes the evergreen stable? Because I really don't want to ship that by default forever

These take the place of `WebView2Loader.dll` in the Win32 APIs, which has a corresponding `WebView2Loader.dll.lib` static import lib. There's also a `WebView2LoaderStatic.lib` which would statically link them into the binary with no need for a separate DLL. Whichever mechanism it used, it was probably coming from the `webview2` crate. At some point, one or both of those files had to be included in the crates, upstream and/or downstream from `wry` to enable the static link step and potentially to include a DLL.

The WinRT API doesn't seem to have the option of linking a static implementation though, so this PR may be introducing the requirement to ship the DLLs with the package. I'll double check what `webview2` was doing here.

Reply:
It looks like `webview2`'s behavior depends on the toolchain: https://docs.rs/webview2/0.1.0-beta.1/webview2/#webview2loader. For `gnu` it dynamically links and you need to include the DLL, for `msvc` it statically links.